### PR TITLE
Iptables purge

### PR DESCRIPTION
**Pull request by swbsf**
Removing and cleaning iptables rules for new and already deployed boxes.

### CODE REVIEW

File: roles/iptables/templates/iptables.rules.j2 [hunk 1]
```diff
@@ -1,77 +0,0 @@
...
-COMMIT
-# Completed on Tue Oct 27 14:37:58 2015
-# Generated by iptables-save v1.4.21 on Tue Oct 27 14:37:58 2015
-*nat
-:PREROUTING ACCEPT [26:2117]
-:INPUT ACCEPT [2:420]
-:OUTPUT ACCEPT [14:4088]
-:POSTROUTING ACCEPT [1:92]
--A POSTROUTING -o {{lan_interface}} -j MASQUERADE
--A POSTROUTING -o {{client_interface}} -j MASQUERADE
```
Comment (fheslouin) [inline roles/iptables/templates/iptables.rules.j2]: We need to keep those rules for the AP

File: roles/iptables/templates/iptables.rules.j2 [hunk 2]
```diff
@@ -1,77 +0,0 @@
...
--A INPUT -p tcp -m tcp --dport 655 -j ACCEPT
--A INPUT -p udp -m udp --dport 655 -j ACCEPT
--A INPUT -p tcp -m tcp --dport 656 -j ACCEPT
--A INPUT -p udp -m udp --dport 656 -j ACCEPT
--A INPUT -p udp -m udp --dport 5353 -j ACCEPT
--A INPUT -p udp -m udp --dport 60000:61000 -j ACCEPT
--A FORWARD -i {{lan_interface}} -o {{hotspot_interface}} -m state --state RELATED,ESTABLISHED -j ACCEPT
--A FORWARD -i {{hotspot_interface}} -o {{lan_interface}} -j ACCEPT
--A FORWARD -i {{client_interface}} -o {{hotspot_interface}} -m state --state RELATED,ESTABLISHED -j ACCEPT
--A FORWARD -i {{hotspot_interface}} -o {{client_interface}} -j ACCEPT
```
Comment (fheslouin) [inline roles/iptables/templates/iptables.rules.j2:57]: Same here, we need to forward the data in order to keep the AP working

File: roles/iptables/templates/iptables.rules.j2 (hunk 1 above)
Comment (swbsf) [reply roles/iptables/templates/iptables.rules.j2]: Nice! I'll add that back.

File: roles/iptables/templates/iptables.rules.j2 (hunk 2 above)
Comment (swbsf) [reply roles/iptables/templates/iptables.rules.j2:57]: Nope, already done, see https://github.com/ideascube/ansiblecube/pull/116/commits/d3a1691f9384f4c924237925e125bab4151e975f#diff-8ed42cfa57d0bf8ab7464fc5c858e976R16.

Comment (fheslouin) [reply roles/iptables/templates/iptables.rules.j2:57]: Oh right !

**Pull request merged**